linux_os/guide/system/software/sudo/sudo_require_reauthentication/rule.yml

```
@@ -26,11 +26,14 @@ severity: medium
 
 identifiers:
     cce@rhel8: CCE-87838-9
+    cce@sle12: CCE-83231-1
     cce@sle15: CCE-85764-9
 
 references:
     disa: CCI-002038
     stigid@rhel8: RHEL-08-010384
+    nist: IA-11
+    stigid@sle12: SLES-12-010113
     stigid@sle15: SLES-15-020102
     srgid: SRG-OS-000373-GPOS-00156
```

Please also fix this line, thank you.
```suggestion
    srg: SRG-OS-000373-GPOS-00156
```

---

Nice one :+1: - missed that. Now should be ok